File: 242-valid-anagram/valid-anagram.js
const isAnagram = function (s, t) {

    if (s.length !== t.length) return false;
    const sObject = {};
    const tObject = {};
    for (let i = 0; i < s.length; i++) {
        sObject[s[i]] = (sObject[s[i]] || 0) + 1;
        tObject[t[i]] = (tObject[t[i]] || 0) + 1;
    }
    for (const key in sObject) {
        if (sObject[key] !== tObject[key]) return false;
    }
    return true;
};

File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t):
            return False
        an=set(t)
        for i in an:
            if s.count(i)!=t.count(i):
                return False
        return True
        